src/printapp/templates/index.html

```diff
@@ -126,6 +135,10 @@ <h3 class="subheader">Welcome, {{email}}</h3>
           <strong>Oops!</strong> We’re having trouble refreshing your print queue. Reload the page to try again. If this does not solve the problem, try signing out and signing back in.
           <a href="#" class="close">&times;</a>
         </div>
+        <div id="deleteError" data-alert hidden class="alert-box alert radius">
+          <strong>Oops!</strong> There was a problem deleting your document. Reload the page then try again. If this does not solve the problem, try signing out and signing back in.
```

Comment:
I honestly have no idea how an error would happen, so I'm not sure what the message should be.